src/tokens/colors/color.stories.jsx

```diff
@@ -52,9 +50,9 @@ const paletteColors = [
 
 export const Color = ({color, name}) => {
     return (
-        <div className={classnames(storyStyles.storyItem)}>
+        <div className={clsx(storyStyles.storyItem)}>
             <p>{name}</p>
-            <div className={classnames(styles.colorShape, cx([`color-${color}`]))}/>
+            <div className={clsx(styles.colorShape, styles[`color-${color}`])}/>
```

Comment:
What is `cx`/`styles` here?

Reply:
`cx` referred to `let cx = classnames.bind(styles);` 
`styles ` referes to `import styles from './colors.stories.module.scss';

It's to load a CSS from a story, it's seems to just import the scss file doesn't work here.

Reply:
Ahh gotcha 👍🏾